test-app/.swcrc

```diff
@@ -1,29 +0,0 @@
-{
```

Comment:
Did you forget to remove the root `.swcrc`?

Reply:
It's still needed for testing. Will remove when I add a jest loader.